skore/src/skore/sklearn/_estimator/metrics_accessor.py

```diff
@@ -486,7 +494,9 @@ def _predict_time(
             "predict_time",
         )
 
-        return self._parent._cache.get(predict_time_cache_key, None)
+        return self._parent._cache.get(
+            predict_time_cache_key, float("nan") if cast else None
```

Comment:
Nitpick: improve the readibility by putting parenthesis.

```suggestion
            predict_time_cache_key, (float("nan") if cast else None)
```